src/core/core.controller.js

```diff
@@ -382,6 +382,10 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 				scales[scale.id] = scale;
 			}
 
+			// parse min/max value, so we can properly determine min/max for other scales
+			scale._userMin = scale._parse(scale.options.ticks.min);
```

Comment:
Would it be more useful to cache `_userBounds` here instead? It seems like we're calling that method in most places which accesses these values, but we could probably just cache the result of the function instead

Reply:
Could not get it to work at first try. I don't think it makes much difference.